internal/exec/shell_utils.go

```diff
@@ -45,6 +45,78 @@ func getNextShellLevel() (int, error) {
 	return shellVal, nil
 }
 
+// ExecuteShellCommandWithPipe prints and executes the provided command with args and flags
+func ExecuteShellCommandWithPipe(
```

Comment:
To pass the output of `terrform plan` to `pipeform`, I wanted to pass stdin and stdout to ExecuteShellCommand, but ExecuteShellCommand doesn't accept them.
So I defined a new function based on ExecuteShellCommand.
We should refactor later.